src/Controller/PdfExportController.php

```diff
@@ -69,8 +69,8 @@ function () use ($phpWordObject) {
      */
     public function render_images($string)
     {
-        $string = str_replace('![alt text](', '<br/><br/><img src ="', $string);
-        $result = str_replace(')', '"/>', $string);
+        $pattern = '/\!\[([^\]]*)alt text\]\(*\((\b(?:(?:https?|ftp):\/\/|www\.)[-a-z0-9+&@#\/%?=~_|!:,.;]*[-a-z0-9+&@#\/%=~_|])\)/';
```

Comment:
This still requires the text for the to have `alt text` at the end of the text, it should be able to be anything.

I would suggest that only the https scheme be allowed (or none specified to allow attachments located on the same server).
  - Using http will make browsers complain about mixed content;
  - and ftp is not supported as of [Chrome 63](https://www.chromestatus.com/feature/5709390967472128) and will not be supported as of [Firefox 61](https://bugzilla.mozilla.org/show_bug.cgi?id=1404744).

So, allowing things of both these forms:
- !\[OpenSALT favicon](https://opensalt.net/favicon32.png)
- !\[GitHub Fluid-Icon](/fluidicon.png)

i.e., something along the lines of `/\!\[([^\]]*)\]\(((?:https:\/\/|\/)[^\)]+)\)/`